index.html

```diff
@@ -5,19 +5,17 @@
 </head>
 
 <body>
-  <hello-world greet="Hoodie" if={showHoodie}></hello-world>
-  <hello-world greet="Titay"></hello-world>
-  <hello-world greet="Hyesoo"></hello-world>
-  <hello-world></hello-world>
-  <contact-list></contact-list>
+  <app></app>
```

Comment:
I moved all the riot tags into a wrapping `app` tag, so that we can work with `if={property}` attributes that Riot provides, but only within tags. When we mount `app`, all tags within it will be mounted automatically